Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1268,11 +1264,7 @@ - (void)processNextQueueItem {
                 [req processResponse:nil error:[NSError errorWithDomain:BNCErrorDomain code:BNCInitError userInfo:@{ NSLocalizedDescriptionKey: @"Branch User Session has not been initialized" }]];
                 return;
             }
-            
-            if (![req isKindOfClass:[BranchCloseRequest class]]) {
-                [self clearTimer];
-            }
-            
+                        
```

Comment:
Won't we need to use that boolean `appIsInBackground` to prevent this code from executing? Here's what I'm imagining:
context: init session network call in progress, 3 items in queue
0. app goes to background
1. new boolean set
2. init session network call returns
3. incorrect: 3 items in queue proceed to execute despite the fact that the app is in the background
expected: 3 items remain outstanding in queue until next session initializes